cmd/kubeadm/app/preflight/checks.go

```diff
@@ -213,9 +215,26 @@ func (hst HttpProxyCheck) Check() (warnings, errors []error) {
 	return nil, nil
 }
 
+type SystemVerificationCheck struct{}
+
+func (sysver SystemVerificationCheck) Check() (warnings, errors []error) {
+	// Create a buffered writer and choose a quite large value (1M) and suppose the output from the system verification test won't exceed the limit
+	bufw := bufio.NewWriterSize(os.Stdout, 1*1024*1024)
```

Comment:
Are you ever actually using the data you're collecting?  If not, it seems like `io.Discard` is what you're looking for.  If you are, why not create a writer which keeps the first and the last and elides the middle if too much is written instead of hoping you have enough space?